js/containers/app.js

```diff
@@ -79,6 +80,7 @@ class App extends PureComponent {
             setMapStatus={setMapStatus} setCrossSectionPoint={setCrossSectionPoint} mark2DViewModified={mark2DViewModified}
             updateEarthquakesData={updateEarthquakesData}
             pins={pins} setPin={setPin} updatePin={updatePin} />
+          <ThumbnailMap crossSectionPoints={crossSectionPoints} mode={mode} layers={layers} />
```

Comment:
I think we could add here a similar condition like one line below: `mode === '3d' &&`. Thumbnail map is a full Leaflet instance. I doubt it'll slow down things in a significant way, but it seems to be an easy enough optimization.